composer.json

```diff
@@ -34,7 +34,7 @@
         "symfony/dependency-injection": "6.1.*",
         "symfony/finder": "^6.2",
         "symfony/string": "^6.2",
-        "symplify/autowire-array-parameter": "^11.1.17",
+        "symplify/autowire-array-parameter": "dev-main as 11.1.17",
```

Comment:
@TomasVotruba this is needed while wait for new release of symplify/autowire-array-parameter so the build will working